hedera-node/hedera-smart-contract-service-impl/src/test/java/module-info.java

```diff
@@ -2,7 +2,17 @@
     requires com.hedera.node.app.service.contract;
     requires com.hedera.node.app.service.contract.impl;
     requires org.junit.jupiter.api;
+    requires com.hedera.node.app.service.mono;
+    requires com.hedera.node.app.service.mono.testFixtures;
+    requires org.mockito;
+    requires org.hamcrest;
+    requires org.assertj.core;
+    requires org.mockito.junit.jupiter;
+    requires com.hedera.node.app.spi;
+    requires hedera.services.hedera.node.hedera.app.spi.testFixtures;
 
     opens com.hedera.node.app.service.contract.impl.test to
             org.junit.platform.commons;
+    opens com.hedera.node.app.service.contract.impl.test.handlers to
+            org.junit.platform.commons;
```

Comment:
Per @hendrikebbers suggestion on a different PR, I _think_ we can use `open module` to get rid of both `opens` lines:

```
open module com.hedera.node.app.service.contract.impl.test {
    requires com.hedera.node.app.service.contract;
    requires com.hedera.node.app.service.contract.impl;
    requires org.junit.jupiter.api;
    requires com.hedera.node.app.service.mono;
    requires com.hedera.node.app.service.mono.testFixtures;
    requires org.mockito;
    requires org.hamcrest;
    requires org.assertj.core;
    requires org.mockito.junit.jupiter;
    requires com.hedera.node.app.spi;
    requires hedera.services.hedera.node.hedera.app.spi.testFixtures;
}

Reply:
I guess it opens module to everything in addition to `org.junit.platform.commons` right ?

Reply:
Yes, but since it's a test module I'm not sure we care. Not a big deal either way though